H265dec

andyLiuChina opened the pull request:
fix SLIST A B C and PICSIZE A B C D

xuguangxin commented on decoder/vaapidecoder_h265.cpp:
```diff
@@ -974,6 +978,8 @@ PicturePtr VaapiDecoderH265::createPicture(const H265SliceHdr* const slice,
         const H265NalUnit* const nalu)
 {
     PicturePtr picture;
+    if(!m_context)
```
in which condition, it will have null context?

andyLiuChina committed:
h265decode: skip RASL_N and RASL_R

andyLiuChina committed:
h265decode: deblocking filter

andyLiuChina committed:
h265parse: get ref_pic

andyLiuChina committed:
h265decode: skip RASL_N and RASL_R

When a CRA picture has NoRaslOutputFlag equal to 1, the associated
RASL pictures are not output by the decoder.(3.24)

andyLiuChina committed:
h265decode: verify whether m_context is NULL

When the resolution is not supported, the m_context created
previously is NULL. If we use a NULL value here, segment error
occured.
